### ISSUE TITLE
Description Incorrect for Example 2 - 1884. Egg Drop With 2 Eggs and N Floors

### ISSUE THREAD

**Issue by stormsunshine** [LeetCode-Feedback/LeetCode-Feedback#5286]
#### Your LeetCode username
stormsunshine

#### Category of the bug
- [x] Question
- [ ] Solution
- [ ] Language
- [ ] Missing Test Cases 

#### Description of the bug
This issue is to reopen #5270 .

For [1884. Egg Drop With 2 Eggs and N Floors](https://leetcode.com/problems/egg-drop-with-2-eggs-and-n-floors), the explanation in example 2 is incorrect. **Please read the following description carefully.**

In example 2, if the 1st egg breaks at floor 9, we know f is between 0 and 8. The 2nd egg should be dropped from floor 1 to floor 8 to determine with certainty what the value of f is, so the 2nd egg should be dropped for 8 times. The sentences should be as follows.

> Drop the 2nd egg starting from floor 1 and going up one at a time to find f within 8 more drops. Total drops is 1 + 8 = 9.

In 1 + 8 = 9, 1 means the 1st egg is dropped once at floor 9 and it breaks, and the 2nd egg is dropped 8 times (from floor 1 to floor 8).

Take the following paragraph as an example.

> If the 1st egg does not break, drop the 1st egg again at floor 22. If it breaks, we know f is between 9 and 21. Drop the 2nd egg starting from floor 10 and going up one at a time to find f within 12 more drops. Total drops is 2 + 12 = 14.

In this case, the 1st egg is dropped twice, at floor 9 and floor 22 respectively, and it breaks at floor 22, and the 2nd egg is dropped 12 times, from floor 10 to floor 21. Therefore, total drops is 2 + 12 = 14.

Since dropping from floor 10 to floor 21 costs 12 drops, dropping from floor 1 to floor 8 costs 8 drops, not 7.

#### Expected behavior
The explanation in example 2 should be updated according to the description above. The sentences after "f is between 0 and 8" should be as follows.

> Drop the 2nd egg starting from floor 1 and going up one at a time to find f within 8 more drops. Total drops is 1 + 8 = 9.

**Comment by LC-Sue** [comment]
Hi @stormsunshine 
Thank you for reaching out to us. I've relayed this issue to our team to investigate.

**Comment by LC-Sue** [comment]
Hi @stormsunshine
Thank you for your time. Our team has looked into the problem and the current description is fine. We need 7 drops for 8 floors because we don't have to try them all. We only need 7 tries and if for the first 7 tries the egg doesn't break and given that it breaks at floor 9, then we know for sure f = 8 without needing a drop.

**Comment by stormsunshine** [comment]
@LC-Sue Have you considered this problem carefully? According the description, f is the maximum floor number such that the egg won't break.

If the egg breaks at floor 9 and does not break at floor 7, do you know whether the egg breaks at floor 8? If the egg breaks at floor 8, then f = 7. If the egg does not break at floor 8, then f = 8. How do you know for sure f = 8 without dropping the egg at floor 8?

If in this case, you only need to drop the egg at floor 7 and floor 9, not floor 8, then in the next case where the egg breaks at floor 22, why do you still need to drop the egg at floor 21 (from floor 10 to floor 21)? This is indeed inconsistent!

In conclusion, to know the exact value of f, you must find the falue f such that both the following requirements are met.
- The egg does not break at floor f.
- The egg breaks at floor f + 1, or f = n.

**Comment by LC-Sue** [comment]
Hi @stormsunshine 
Thank you for reaching out. I have relayed your feedback over to our team again for consideration!

**Comment by LC-Sue** [comment]
Hi @stormsunshine,
Thank you for your time, our content time has looked at the problem again and updated the problem!
